src/lasuite/oidc/backends.py

```diff
@@ -0,0 +1,178 @@
+"""Authentication Backends for the People core app."""
```

Comment:
You mention the people project here. Probably it is present in other module, you can probably do a global check.